ament_cmake_gmock/ament_cmake_gmock-extras.cmake

```diff
@@ -36,6 +42,7 @@ macro(_ament_cmake_gmock_find_gmock)
         NO_DEFAULT_PATH NO_CMAKE_FIND_ROOT_PATH
       )
       find_file(_gmock_src_file "gmock.cc"
+        NAMES "gmock-gtest-all.cc"  # Alternative when using "fused" sources.
```

Comment:
Good point, I didn't notice the "or" part, such a silly API. Fixed in 8d02524.

This needs to be tested on the farm, here are some CI jobs:

- http://ci.ros2.org/job/ros2_batch_ci_linux/222/
- http://ci.ros2.org/job/ros2_batch_ci_osx/146/
- http://ci.ros2.org/job/ros2_batch_ci_windows/226/

I'll make sure to look through the output of CMake in the Linux job to make sure it's getting gmock from `/usr/src/...`.